CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## UNRELEASED
 
+## v2.0.0
+
+### Changed
+
+- Updated the return type annotation of `NewRelicInteractorInterface::disableAutoRUM` to `?bool` 
+to match the latest changes in the NewRelic API. 
```

Comment:
shouldn't it be
```
- Update ...
  to match ...
```

(align multi line text with bullet)

Reply:
Thank you